src/games/strategy/triplea/ai/AdvancedUtils.java

```diff
@@ -36,24 +36,6 @@ public static int getIndexOfLastUnitMatching(final List<Unit> units, final Match
     return -1;
   }
 
-  public static Unit getFirstUnitMatching(final List<Unit> units, final Match<Unit> match, final int startIndex) {
```

Comment:
Honestly, if none of the AIs are using them at this point then I'd say just remove them.